zero/helm/Chart.yaml

```
@@ -0,0 +1,18 @@
+apiVersion: v2
+name: pomerium-zero
+description: A Helm chart for deploying Pomerium Zero
+type: application
+version: 0.27.2
+appVersion: "v0.27.2"
+home: https://www.pomerium.com/
+icon: https://www.pomerium.com/static-img/favicon.svg
+sources:
+  - https://github.com/pomerium/pomerium
+maintainers:
+  - name: Pomerium Team
+    email: info@pomerium.com
```

```suggestion
    email: support@pomerium.com
```

I think info goes to sales